comment-service/src/main/java/com/in2it/commentandlikeservice/response/Response.java

```diff
@@ -0,0 +1,21 @@
+package com.in2it.commentandlikeservice.response;
+
+import java.time.LocalDateTime;
+
+import org.springframework.http.HttpStatus;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Response<T> {
+
+	private T data;
+	private String message;
+	private HttpStatus httpStatus;
+	private int statusCode; 
+	private LocalDateTime timeStamp;

```

Comment:
adding this field is every response will not have any effect as it is going to set VM's time.